modules/touch/src/touchinteraction.cpp

```diff
@@ -678,100 +843,379 @@ void TouchInteraction::findSelectedNode(const std::vector<TuioCursor>& list) {
                 }
             }
         }
-    }
+    }
+
+    // If an item has been picked, it's in the first position of the vector now
+    if (SceneGraphNode* node = std::get<0>(currentlyPicked)) {
+        _pickingSelected = node;
+#ifdef TOUCH_DEBUG_NODE_PICK_MESSAGES
+        LINFOC("Picking", "Picked node: " + _pickingSelected->identifier());
+#endif //#ifdef TOUCH_DEBUG_NODE_PICK_MESSAGES
+    }
+
+    _selected = std::move(newSelected);
+}
+void TouchInteraction::findSelectedNode(const std::vector<TouchInputs>& list) {
+    //trim list to only contain visible nodes that make sense
+    std::string selectables[30] = {
+        "Sun", "Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus",
+        "Neptune", "Pluto", "Moon", "Titan", "Rhea", "Mimas", "Iapetus", "Enceladus",
+        "Dione", "Io", "Ganymede", "Europa", "Callisto", "NewHorizons", "Styx", "Nix",
+        "Kerberos", "Hydra", "Charon", "Tethys", "OsirisRex", "Bennu"
+    };
```

Comment:
This list should go in a config file somewhere, eventually

Reply:
I opened an issue about this - see #1041